packages/console/src/pages/SignInExperience/PageContent/utils/parser.ts

```diff
@@ -176,6 +178,7 @@ export const signInExperienceToUpdatedDataParser = (
     passwordPolicy,
     captchaPolicy,
     sentinelPolicy,
```

Comment:
I'm not sure, but aren't the password policy, captcha and sentinel moved to "Security" already? Are they still used in SIE settings?